torch/lib/THC/generic/THCTensorRandom.cu

```diff
@@ -457,6 +466,69 @@ THC_API void THCTensor_(geometric)(THCState* state, THCTensor *self_, double p)
 
   THCTensor_(freeCopyTo)(state, self, self_);
 };
+
+THC_API void THCTensor_(clampedRandom)(THCState* state, THCTensor *self_, int64_t min_val, int64_t max_val)
+{
+  THArgCheck(min_val < max_val, 2,
```

Comment:
nit: this double cast in the message might overflow. Since random always returns integers, maybe let's print them as long longs?